src/Chalkboard/Chalkboard.js

```diff
@@ -91,6 +91,7 @@ class Chalkboard extends React.Component {
           currentRound={this.props.currentRound}
           score={this.props.score}
         />
+        <progress className='nes-progress is-pattern' value={this.state.answerCount} max='30'/>
```

Comment:
So cool you got the progress bar working!

Reply:
Stoked on the progress bar!